using-kubecost/navigating-the-kubecost-ui/efficiency.md

```diff
@@ -0,0 +1,45 @@
+# Efficiency Report
+
+Efficiency Report is a Monitoring dashboard which provides workload and infra idle costs for your clusters. Metrics include consumption cost of individual CPU/RAM/storage metrics, and total workload efficiency.
+
+Efficiency Report supports clusters by all major cloud service providers, and on-prem clusters.
+
+![Efficiency Report](/images/efficiency.png)
+
+## Understanding effiency and idle cost metrics
+
+Efficiency Report breaks down idle costs into two distinct categories:
+
+* *Workload idle* refers to hardware resources (CPU/GPU/RAM) that are requested by the workloads, but are not consumed on average.
+* *Infra idle* refers to purchased hardware resources that are neither requested nor consumed.
+
+These metrics, when compared to total spend, are necessary when calculating efficiency. Workload efficiency can be calculated as:
+
+> ((resources consumed/resources requested) * 100)

```

Comment:
Do we need an equivalent section explaining the math for Cluster Efficiency?

Reply:
Yes, just realized there are two separate efficiency statistics offered. Can we assume the Cluster Efficiency stat on this page will match the efficiency found in the Clusters page?

Reply:
We talked about this in Slack but NOPE. That page has Workload Efficiency as of this moment.